be/src/exec/vectorized/dict_decode_node.cpp

```diff
@@ -0,0 +1,99 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "exec/vectorized/dict_decode_node.h"
+
+#include "column/chunk.h"
+#include "column/column_helper.h"
+#include "runtime/runtime_state.h"
+#include "util/runtime_profile.h"
+
+namespace starrocks::vectorized {
+
+DictDecodeNode::DictDecodeNode(ObjectPool* pool, const TPlanNode& tnode, const DescriptorTbl& descs)
+        : ExecNode(pool, tnode, descs), _decode_node(tnode.decode_node) {}
+
+Status DictDecodeNode::init(const TPlanNode& tnode, RuntimeState* state) {
+    RETURN_IF_ERROR(ExecNode::init(tnode, state));
```

Comment:
It was not a good idea, init decoder in init stage, I suggest move it to prepare stage